.github/workflows/daily-generate.yml

```diff
@@ -12,13 +12,15 @@ on:
           - prime1
           - prime2
           - samus_returns
+          - am2r
 
   schedule:
     - cron: '0 16 * * 3'  # Cave Story
     - cron: '0 23 * * *'  # Dread
     - cron: '0 16 * * 2'  # Prime 1
     - cron: '0 16 * * 1'  # Prime 2
     - cron: '0 16 * * 4'  # Samus Returns
+    - cron: '0 10 * * 1,5' # AM2R
```

Comment:
Is 1,5 going to cause issues? I know it's a regional thing but I am curious.

Reply:
See the image posted in the PR description :)